pyseir/load_data.py

```diff
@@ -132,17 +132,19 @@ def cache_public_implementations_data():
     df.to_pickle(os.path.join(DATA_DIR, "public_implementations_data.pkl"))
 
 
-@lru_cache(maxsize=32)
-def load_new_case_data_by_fips(
-    fips, t0, include_testing_correction=False, testing_correction_smoothing_tau=5
+def calc_new_case_data_by_region(
```

Comment:
nit: would prefer `calculate` over `calc`, but understand its more verbose

Reply:
Done since it didn't create lots of new lines.